tinydb/naivetable/naivetable_test.go

```diff
@@ -0,0 +1,68 @@
+package naivetable
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"github.com/rob-sokolowski/go-db-tutorial/tinydb"
+	"math"
+
+	"testing"
+)
+
+func generateRandomString(min, max int, mu float64) string {
+	// randNormFloat64 generates a normally distributed float64 in range (-1, 1)
+	randNormFloat64 := func() float64 {
+		var u uint64
+		_ = binary.Read(rand.Reader, binary.BigEndian, &u)
+		return 2*(float64(u)/float64(1<<64)) - 1
```

Comment:
- BigEndian? Is this just specifying to Reader how to read the binary?
- &u? - Is this math stuff?